Marlin/configuration_store.cpp

```diff
@@ -58,7 +58,7 @@
  *  188  M206 XYZ  home_offset (float x3)
  *
  * Mesh bed leveling:
- *  200  M420 S    active (bool)
+ *  200  M420 S    status (0 = empty, 1 = has mesh numbers)
```

Comment:
Comments in this header include the data-types as cues to the size of each field. We don't really need usage info here. So you can simply change `bool` to `char` or `uint8`.